i18n/ko.json

```
@@ -1,1962 +1,1962 @@
+    "reset_settings_to_default": "설정을 기본값으로 초기화",
+    "reset_settings_to_recent_saved": "설정을 최근 저장된 상태로 복원",
+    "scanning_library": "라이브러리 스캔 중",
+    "search_jobs": "작업 검색…",
+    "send_welcome_email": "환영 이메일 보내기",
+    "server_external_domain_settings": "외부 도메인",
+    "server_external_domain_settings_description": "공유 링크에 사용할 도메인 주소 (http(s):// 포함)",
+    "server_public_users": "공개 사용자 목록",
+    "server_public_users_description": "공유 앨범에 사용자를 추가할 때 모든 사용자(이름, 이메일)를 표시합니다. 비활성화 시 관리 사용자만 목록에 접근할 수 있습니다.",
+    "server_settings": "서버 설정",
+    "server_settings_description": "서버 관련 설정 관리",
+    "server_welcome_message": "환영 메시지",
+    "server_welcome_message_description": "로그인 페이지에 표시되는 메시지입니다.",
+    "sidecar_job": "사이드카 메타데이터",
+    "sidecar_job_description": "파일 시스템에서 사이드카 메타데이터를 검색하거나 동기화합니다.",
+    "slideshow_duration_description": "각 이미지를 표시할 시간 (초)",
+    "smart_search_job_description": "기계 학습을 통해 스마트 검색을 위한 데이터 처리",
+    "storage_template_date_time_description": "항목의 생성 시간을 날짜/시간 정보로 사용합니다.",
+    "storage_template_date_time_sample": "예시 시간 {date}",
+    "storage_template_enable_description": "저장 템플릿 엔진 활성화",
+    "storage_template_hash_verification_enabled": "해시 검증 활성화됨",
+    "storage_template_hash_verification_enabled_description": "해시 검증을 활성화합니다. 영향에 대해 확실하지 않다면 비활성화하지 마세요.",
+    "storage_template_migration": "저장 템플릿 마이그레이션",
+    "storage_template_migration_description": "현재 <link>{template}</link> 템플릿을 기존 업로드 항목에 적용합니다.",
+    "storage_template_migration_info": "이 템플릿은 확장자를 모두 소문자로 변환합니다. 변경 사항은 새 항목에만 적용되며, 기존 항목에 소급 적용하려면 <link>{job}</link> 작업을 실행하세요.",
+    "storage_template_migration_job": "저장 템플릿 마이그레이션 작업",
+    "storage_template_more_details": "이 기능에 대한 자세한 내용은 <template-link>저장 템플릿</template-link>과 <implications-link>영향</implications-link> 문서를 참조하세요.",
+    "storage_template_onboarding_description": "이 기능을 활성화하면 사용자 정의 템플릿에 따라 파일이 자동 정리됩니다. 안정성 문제로 인해 기본적으로 꺼져 있습니다. 자세한 내용은 <link>문서</link>를 참조하세요.",
+    "storage_template_path_length": "예상 경로 길이 제한: <b>{length, number}</b>/{limit, number}",
+    "storage_template_settings": "저장 템플릿",
+    "storage_template_settings_description": "업로드 항목의 폴더 구조 및 파일 이름 설정 관리",
+    "storage_template_user_label": "<code>{label}</code>은 사용자의 스토리지 라벨입니다.",
+    "system_settings": "시스템 설정",
+    "tag_cleanup_job": "태그 정리",
+    "template_email_available_tags": "템플릿에서 다음 변수를 사용할 수 있습니다: {tags}",
+    "template_email_if_empty": "템플릿이 비어 있으면 기본 이메일이 사용됩니다.",
+    "template_email_invite_album": "앨범 초대 이메일 템플릿",
+    "template_email_preview": "미리보기",
+    "template_email_settings": "이메일 템플릿",
+    "template_email_settings_description": "사용자 정의 이메일 알림 템플릿 관리",
+    "template_email_update_album": "앨범 업데이트 이메일 템플릿",
+    "template_email_welcome": "환영 이메일 템플릿",
+    "template_settings": "알림 템플릿",
+    "template_settings_description": "사용자 정의 알림 템플릿 관리",
+    "theme_custom_css_settings": "사용자 정의 CSS",
+    "theme_custom_css_settings_description": "CSS를 통해 Immich의 디자인을 커스터마이즈할 수 있습니다.",
```

```suggestion
    "theme_custom_css_settings_description": "CSS를 통해 Immich의 디자인을 커스터마이징할 수 있습니다.",
```
Since the term “커스터마이징” is more commonly used in contexts like games and DIY projetc in Korea, using “커스터마이징” here feels more natural.

---

음차하지 않고 다른 표현으로 바꾸는 것이 더 적절해보입니다.